pkg/workloads/cortex/downloader/download.py

```diff
@@ -50,14 +53,17 @@ def start(args):
                 src = os.path.join(dir_path, entries[0])
                 os.rename(src, dest)
 
+    if download_config.get("last_log", "") != "":
```

Comment:
`if download_config.get("last_log") is not None:` is the pythonic way of checking key existence (I know there are other instances of it in this file)